portiaui/app/components/list-item-icon.js

```diff
@@ -1,5 +1,7 @@
 import IconButton from './icon-button';
 
 export default IconButton.extend({
-    classNames: ['list-item-icon']
+    classNames: ['list-item-icon'],
+    classNameBindings: ['hasIndent'],
+    hasIndent: false
```

Comment:
i think that instead of the icon being responsible for indent we should have a spacer component that is used in the template when there is no badge.